### ISSUE TITLE
Python error 'int' object has no attribute 'shape'

### ISSUE THREAD

**Issue by MritulaC** [jacobgil/keras-dcgan#31]
I kept getting an error " Python error 'int' object has no attribute 'shape'" reflecting to the following line  from def train in the dcgan.py file

`g_loss = d_on_g.train_on_batch(noise, [1] * BATCH_SIZE))`

  Solution: 
Changing the line to the below worked well:
`g_loss = d_on_g.train_on_batch(noise,np.array([1] * BATCH_SIZE))`

Correct me if this is wrong!

Hope this is of some help!

Thanks,
Mritula

**Comment by Kang-Jack** [comment]
yes, fixed issue on my machine, in same file  line ‘d_loss = d.train_on_batch(X, y)’ need be modified to 'd_loss = d.train_on_batch(X, np.array(y))' as well. thx
